mungegithub/mungers/publisher.go

```diff
@@ -290,12 +312,29 @@ Repos:
 			}
 		}
 	}
-	return errors.NewAggregate(errlist)
+
+	// create an issue or update an existing issue to report the failed
+	// publisher run
+	aggregated := errors.NewAggregate(errlist)
+	if len(errlist) != 0 {
+		glog.Flush()
+		// maxLogLength is the estimated number of characters of the log created
+		// in each run of the publisher
+		var maxLogLength = int64(15000)
```

Comment:
Alternatively, we could have some marker line and search for that. Or is there some log rotation in glog?